options/options.go

```diff
@@ -38,15 +38,17 @@ type Options struct {
 	RubyGems   string
 	PolicyFile string
 	// TODO(action): Add logic for writing results to file
-	ResultsFile string
-	ChecksToRun []string
-	Metadata    []string
-	ShowDetails bool
+	ResultsFile    string
+	ChecksToRun    []string
+	Metadata       []string
+	ShowDetails    bool
+	Dependencydiff string
 
 	// Feature flags.
-	EnableSarif       bool `env:"ENABLE_SARIF"`
-	EnableScorecardV5 bool `env:"SCORECARD_V5"`
-	EnableScorecardV6 bool `env:"SCORECARD_V6"`
+	EnableSarif          bool `env:"ENABLE_SARIF"`
+	EnableScorecardV5    bool `env:"SCORECARD_V5"`
+	EnableScorecardV6    bool `env:"SCORECARD_V6"`
+	EnableDependencydiff bool `env:"ENABLE_DEPENDENCYDIFF"`
```

Comment:
shall we start using a common `SCORECARD_EXPERIMENTAL` flag? I think this is something that was proposed before to simplify. (Don't rename the other ones, we can start using it for new features only)

@naveensrinivasan @azeemsgoogle Wdut?

Reply:
Yes, please. Let's not add more env vars to gate experimental features.